app/Http/Controllers/Admin/CallchampionsController.php

```diff
@@ -41,6 +41,7 @@ public function __construct(){
 			$this->user_id=$userinfo['user_id'];
 			$this->role_id=$userinfo['role_id'];
 			$this->role_type=$userinfo['v_role'];
+			$this->i_phone_number=$userinfo['i_phone_number'];
```

Comment:
Causes a bug for non-admin logins.
This info is not stored there.
Why is this needed?